subsys/bluetooth/services/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 #
-# Copyright (c) 2018 Nordic Semiconductor
+# Copyright (c) 2018 - 2020 Nordic Semiconductor ASA
```

Comment:
I think you don't need to modify this year here

Reply:
Done.